attestation.md

```diff
@@ -0,0 +1,89 @@
+# 🔖 Attestation 
+
+In this section, you'll play the role of an <span class="label-role attester">attester</span>:
```

Comment:
Instead of "you'll play the role of an attester", it might be better "you'll play the role of the attester".

They are both grammatically correct, but it sounds better as you are declaring for the reader their role in this section.